### PR TITLE
#7656 Fix missing learn more link

### PR DESCRIPTION
**Pull request by levdavid**

### CODE REVIEW

**Review by dariomws — APPROVED** [review]

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward Please review. Thanks

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Kateyanne Please merge this PR. Thanks

**Comment by Kateyanne** [comment]
checking on the link @yogkumgit

File: teams/teams-ps/teams/New-CsTeamTemplate.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
-This cmdlet lets you provision a new team template for use in Microsoft Teams. Learn more about team templates here.
+This cmdlet lets you provision a new team template for use in Microsoft Teams. Learn more about team templates [here](https://docs.microsoft.com/MicrosoftTeams/get-started-with-teams-templates-in-the-admin-console).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamTemplate.md:17]:
```suggestion
This cmdlet lets you provision a new team template for use in Microsoft Teams. To learn more about team templates, see [Get started with Teams templates in the admin center](MicrosoftTeams/get-started-with-teams-templates-in-the-admin-console).

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - Link changed as per hyperlink protocol.

File: teams/teams-ps/teams/New-CsTeamTemplate.md [hunk 2]
```diff
@@ -14,7 +14,8 @@ schema: 2.0.0
 
 ## SYNOPSIS
 
-This cmdlet lets you provision a new team template for use in Microsoft Teams. Learn more about team templates here.
+This cmdlet lets you provision a new team template for use in Microsoft Teams. To learn more about team templates, see [Get started with Teams templates in the admin center](MicrosoftTeams/get-started-with-teams-templates-in-the-admin-console).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamTemplate.md:17]:
```suggestion
This cmdlet lets you provision a new team template for use in Microsoft Teams. To learn more about team templates, see [Get started with Teams templates in the admin center](https://docs.microsoft.com/MicrosoftTeams/get-started-with-teams-templates-in-the-admin-console).

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @levdavid - the site-relative link does not seem to work in this instance. Sorry about the hassle, but could you please change it again? Many thanks.

**Comment by levdavid** [comment]
@JohanFreelancer9 I have changed the link back, should I revert my commit? Wasn't sure if the relative link ended up working 😅

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JohanFreelancer9** [comment]
@levdavid - Perfect, thank you. It is fine now.

**Comment by Kateyanne** [comment]
@levdavid @JohanFreelancer9  After more research, we do want to fix the link. It will work if there is a / in front of MicrosoftTeams. So the correct link is  /MicrosoftTeams/get-started-with-teams-templates-in-the-admin-console. Sorry for the churn - could you update the link?

**Comment by levdavid** [comment]
@Kateyanne updated!

**Pull request merged**